static-site/content/community-datasets.yaml

```diff
@@ -208,3 +208,7 @@ data:
     maintainers: Grupo Inmunovirologia, facultad de medicina, Universidad de Antioquia, Medellin, Colombia
     date: '2022-05-18'
     title: Genomic epidemiology of SARS-CoV-2 in Colombia
+  - url : https://nextstrain.org/community/KelianP/RYMV-WA/RYMV/WA
```

Comment:
Hello @KelianP, thank you for sharing your community dataset! 

Currently, your community dataset is not viewable because the dataset filename ([RYMV_WA.json](https://github.com/KelianP/RYMV-WA/blob/b300015e48ac4f48a4aab6986e906b71086a3154/auspice/RYMV_WA.json)) does not begin with the exact repo name (see [community docs](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html) for more details). 

If you update the dataset filename to "RYMV-WA.json" by changing the underscore to a dash, then your dataset should be viewable at the URL https://nextstrain.org/community/KelianP/RYMV-WA

Reply:
Hello @joverlee521, thank you very much for your message!
I just changed the dataset filename in my repository and our dataset is now viewable at the URL [dataset should be viewable at the URL https://nextstrain.org/community/KelianP/RYMV-WA]

Reply:
Thanks for updating @KelianP. 
You should be able to see your community build listed at https://nextstrain.org/community/

Reply:
Thanks @joverlee521 for your help.
It was a pleasure to work on a new methodology to display a continuous phylogeography (using a MCC BEAST entry) on Nextstrain.